sentry-graphql/src/main/java/io/sentry/graphql/NoOpSubscriptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package io.sentry.graphql;
+
+import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
+import io.sentry.IHub;
+
+public final class NoOpSubscriptionHandler implements SentrySubscriptionHandler {
```

Comment:
Should this be annotated as internal?

Reply:
Depending on how we solve the `SentryInstrumentation` ctor mess, this may be a valid choice for users to pass in so I think it should be public.